client/src/components/PropertiesSummary.js

```diff
@@ -8,15 +8,11 @@ import APIClient from "components/APIClient";
 import "styles/PropertiesSummary.css";
 import { EvictionsSummary } from "./EvictionsSummary";
 import { RentstabSummary } from "./RentstabSummary";
+import { ViolationsSummary } from "./ViolationsSummary";
```

Comment:
Nice! Thank you for refactoring this piece of the Summary tab into its own component— it's a lot easier to follow along that way.